src/PrestaShopBundle/Resources/config/routing/admin/improve/design/_mail_theme_alias.yml

```diff
@@ -0,0 +1,22 @@
+# These are alias routes used to manage legacy urls that were badly formatted (camel case)
+_admin_mail_theme_save_configuration_alias:
+  path: /saveConfiguration
+  defaults:
+    _controller: FrameworkBundle:Redirect:redirect
```

Comment:
is `_controller` mandatory if you add the `route` param?

Reply:
Yes it's actually this controller (from the framework) that performs the redirection, using the provided route parameter